packages/compass-e2e-tests/tests/collection-indexes-tab.test.ts

```diff
@@ -88,4 +95,73 @@ describe('Collection indexes tab', function () {
 
     await indexComponent.waitForDisplayed({ reverse: true });
   });
+
+  describe('server version 6.1.0 with env variable COMPASS_COLUMNSTORE_INDEXES = true', function () {
+    let initialEnvVars: NodeJS.ProcessEnv;
+
+    before(function () {
+      initialEnvVars = Object.assign({}, process.env);
+
+      process.env.COMPASS_COLUMNSTORE_INDEXES = 'true';
+    });
+
+    after(function () {
+      process.env = initialEnvVars;
+    });
+
+    it('supports creating a columnstore index', async function () {
+      if (semver.lt(MONGODB_VERSION, '6.1.0')) {
```

Comment:
we can maybe try to include alpha or rcs as well so we'll find out sooner. I think it should be possible to just add -alpha or something like here https://github.com/mongodb-js/compass/blob/da327e148b3c3d763af88eac7c4e385c67958779/packages/databases-collections/src/components/collection-fields/has-fle2-support.js#L3

Reply:
That will only help if we ever actually run these tests against alphas or rcs, but might also make it easier to run your test manually. by specifying MONGODB_VERSION so mongodb runner installs something that supports it.